src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -1001,5 +1051,71 @@ private async ValueTask<bool> HandleSetStartCounter(ClientPlayer sender, int seq
 
             return true;
         }
+
+        private async ValueTask<bool> HandleCheckVanish(ClientPlayer sender, float maxDuration)
+        {
+            if (!await ValidateRole(RpcCalls.CheckVanish, sender, PlayerInfo, RoleTypes.Phantom))
```

Comment:
ValidateRole got used twice in these checks

Reply:
Good point, fixed